Misc/NEWS.d/next/Library/2023-04-29-20-49-13.gh-issue-104003.-8Ruk2.rst

```diff
@@ -0,0 +1,2 @@
+Add :func:`warnings.deprecated`, a decorator to mark deprecated functions to
+static type checkers. See :pep:`702`. Patch by Jelle Zijlstra.
```

Comment:
Again, I think it would be good to mention that the decorator has a runtime effect as well as a "statictime" effect